Apps/Playground/X11/App.cpp

```diff
@@ -239,16 +239,46 @@ int main(int _argc, const char* const* _argv)
                 case ButtonPress:
                     {
                         const XMotionEvent& xmotion = event.xmotion;
+                        const XButtonEvent& xbutton = event.xbutton;
+
                         if (nativeInput != nullptr) {
-                            nativeInput->MouseDown(0, xmotion.x, xmotion.y);
+                            switch (xbutton.button) {
+                                case Button1:
+                                    nativeInput->MouseDown(Babylon::Plugins::NativeInput::LEFT_MOUSE_BUTTON_ID, xmotion.x, xmotion.y);
+                                    break;
+                                case Button2:
+                                    nativeInput->MouseDown(Babylon::Plugins::NativeInput::MIDDLE_MOUSE_BUTTON_ID, xmotion.x, xmotion.y);
+                                    break;
+                                case Button3:
+                                    nativeInput->MouseDown(Babylon::Plugins::NativeInput::RIGHT_MOUSE_BUTTON_ID, xmotion.x, xmotion.y);
+                                    break;
+                                case Button4:
+                                    nativeInput->MouseWheel(Babylon::Plugins::NativeInput::MOUSEWHEEL_Y_ID, -120);
```

Comment:
Just curious. Is 120 a specific number that matches browser behavior?

Reply:
That is the current value of deltaY when you trigger a wheel event in Chrome.